setup/basic.md

```diff
@@ -279,12 +361,23 @@ turtlebot4-setup
 
 Go to <b>Wi-Fi Setup</b> and select <b>Apply Defaults</b>. Optionally you can set your own SSID and password before saving and applying the new settings.
 
+{% endtab %}
+{% tab ap jazzy %}
+
+SSH into the Raspberry Pi and run the [TurtleBot 4 setup tool](../software/turtlebot4_setup.md#configuration-tools):
+
+```bash
+turtlebot4-setup
+```
+
+Go to <b>Wi-Fi Setup</b> and select <b>Apply Defaults</b>. Optionally you can set your own SSID and password before saving and applying the new settings.
+
 {% endtab %}
 {% endtabs %}
 
 ```tip
-If you are moving your TurtleBot 4 to a new location with a different Wi-Fi network, 
-reconfigure the Raspberry Pi to connect to that network beforehand or place it into AP mode. 
+If you are moving your TurtleBot 4 to a new location with a different Wi-Fi network,
+reconfigure the Raspberry Pi to connect to that network beforehand or place it into AP mode.
```

Comment:
Do we want to force newlines like that though? For adaptive screen sizes or users with accessibility issues (e.g. larger-than-default fonts) it seems better to just let the default line-wrapping do its thing, rather than manually inserting line breaks that may not always align correctly.

Reply:
Some of the trailing spaces removed were already at the ends of paragraphs or were independent list items, so I've left those alone.  The rest I've removed the superfluous line breaks from the source.

Reply:
I agree with this choice.

Reply:
I am marking this as resolved but I have flagged several other locations where the same issue needs to be addressed with the same resolution.